Add release notes for 1.12.5 (#81)
anbox-cloud/discourse-docs: announcements/1.12.5.md

### Introduction

The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.12.5.

Please see [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.

#### New features & improvements

n/a

#### Bugs

* AC-676 Launching containers on a specific node fails with "node not found"
* AC-671 Application that was signed with a custom system image  can't access hidden APIs
* AC-667 Multi touch is not working when streaming from webrtc platform

> * AC-663 Latest firefox doesn't render in 1.12.x
> * AC-662 Android containers fails to start on Linux 5.13.x

Reviewer: ```suggestion
* AC-663 Latest Firefox doesn't render in 1.12.x
* AC-662 Android containers fail to start on Linux 5.13.x
```

Reviewer: Usually I don't suggest fixes for the bug descriptions, but I feel these must be fixed.

### Introduction

The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.12.5.

> This is an out of order release to prepare for an upcoming Ubuntu kernel release on AWS. With the 5.13 kernel, which will be introduced on instances running Ubuntu 20.04 (focal) starting end of February, Anbox needs to be adopted to work with changes in the `/proc` file system and semantic changes to the mount system API. If you are running on Ubuntu with a kernel version greater or equal than 5.13 please ensure you're using the 1.12.5 images. Using older images will result in Anbox and the nested Android container failing to start.

Reviewer: ```suggestion
This release prepares for an upcoming Ubuntu kernel release on AWS. With the 5.13 kernel, which will be introduced on instances running Ubuntu 20.04 (focal) starting at the end of February, Anbox must be adopted to work with changes in the `/proc` file system and semantic changes to the mount system API. If you are running on Ubuntu with kernel version 5.13 or later, make sure you're using the 1.12.5 images. Using older images will result in Anbox and the nested Android container failing to start.
```

Reviewer: Do we need the "out of order"? This is kinda misleading since it usually means "broken". ;)
If it is required, we could go for "unplanned" or "unscheduled", but I don't think it's really needed.

> Instances running Ubuntu 18.04 (bionic) are not affected as the maximum kernel version is 5.4.

Reviewer: ```suggestion
Instances running Ubuntu 18.04 (bionic) are not affected, because the latest kernel version for Ubuntu 18.04 is 5.4.
```
